File: modul2/Python/PRAK205-2310817210029-Putra_Whyra_Pratama_Setiawan.py
import math
tinggi, sisimiring = input("").split(" ")
alas = round(math.sqrt(pow(int(sisimiring), 2)-pow(int(tinggi), 2)))
keliling = round(int(tinggi)+int(sisimiring)+int(alas))
luas = round((int(tinggi)*int(alas))/2)
print(f"Alas = {alas} cm") 
print(f"Keliling = {keliling} cm") 
print(f"Luas = {luas} cm^2") 

File: modul2/C/PRAK205-2310817210029-Putra_Whyra_Pratama_Setiawan.c
#include <stdio.h>
#include <math.h>
void main() {
    int a, b, c, luas, keliling;
    scanf("%d %d", &a, &b);
    c = sqrt(pow(b, 2)-pow(a, 2));
    keliling = a+b+c;
    luas = (a*c)/2;
    printf("Alas: %d cm\n", c);
    printf("Tinggi: %d cm\n", a);
    printf("Keliling: %d cm\n", keliling);
    printf("Luas: %d cm^2\n", luas);
}